paper.md

```diff
@@ -42,65 +42,69 @@ Born of idea to provide a unified data distribution for neuroscience, taking a v
 Code, data, and computing environments are at the core of scientific practice, and unobstructed access and efficient management of all those digital objects promotes scientific discovery through collaboration, reproducibility, and replicability. 
 While code management and sharing is streamlined with the advance of software distributions, distributed version control systems, and social coding portals like GitHub, data has remained a “2nd-class citizen” in the contemporary scientific process, despite FAIR principles postulating demands on public data hosting portals and the big callout for Data Science.
 Disconnected FAIR data hosting portals provide a cacophony of data access and authentication methods, data versioning is frequently ignored, and shared data provenance is often not recoverable simply because data management is rarely considered to be an integral part of the scientific process.
+DataLad aims to solve these issues by streamlining data consumption, publication, and updating routines, by providing a simplified core API for git and git-annex operations, and by providing additional features for reproducible science such as command execution and automatic recomputation based on complete process provenance.
+As it is interoperable with a large variety of scientific and commercial hosting services and available for all operating systems, DataLad can be integrated into established systems with minimal adjustments.
 
 ## Why git and git-annex
 
-Git is an excellent distributed content management system geared toward management and collaboration on text files, and as such cannot handle large or binary files well.
```

Comment:
> Git is an excellent distributed content management system

Probably best to stick to general phrasing in scientific papers. Unless there is a citation or a solid argument why it is excellent, this is not the place to make such claims IMHO